ledger/apptxn_test.go

```diff
@@ -353,7 +353,7 @@ func TestClawbackAction(t *testing.T) {
 	t.Parallel()
 
 	genBalances, addrs, _ := ledgertesting.NewTestGenesis()
-	l := newTestLedger(t, genBalances)
+	l := newTestFilteredLedger(t, genBalances)
```

Comment:
All these seem very repetitive. Should `newTestLedger` be changed to filter? It seems hard to give good advice for someone to use one sometimes and another elsewhere?